fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -781,6 +801,10 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = PREFER_COMPUTE_NODE)
     private boolean preferComputeNode = false;
 
+    @VariableMgr.VarAttr(name = COMPUTATION_FRAGMENT_SCHEDULING_POLICY)
+    private ComputationFragmentSchedulingPolicy computationFragmentSchedulingPolicy = 
```

Comment:
need some simple test to ensure the session variable can be set via string value. Either through MockCluster or through the sql-tester framework.

Reply:
How to use the sql-tester framework to check it ? Could you please give some tip ?

Reply:
check the content under https://github.com/StarRocks/starrocks/tree/main/test. create new test cases running sql queries and validate the result.

Reply:
Need to validate the user set value in `SetStmtAnalyzer::analyzeSystemVariable` and add some UT in `AnalyzeSetVariableTest`.

Reply:
Its type should be `String`, because the type of session variables cannot be `enum`. See `VariableMgr::setValue` for detail.

Reply:
Could refer to the implementation of the session variable `choose_execute_instances_mode`, since it is also treated as an `enum`.

Reply:
ok，i will update later

Reply:
@kevincai @ZiheLiu i have updated the pr according to your suggestion , please approve the workflow and review again.

Reply:
@kevincai @ZiheLiu It failed in part of AnalyzeSetVariableTest.java in FE UT. Becase it not contain IllegalArgumentException in the analyzeFail function. i add it. Could you please approve the workflow and review again ?